### ISSUE TITLE
Assasin (sin) rogue - behaves like you are using indiscriminated carnage (or target swap) when both are disabled

### ISSUE THREAD

**Issue by Squatty1** [Hekili/hekili#4070]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Spec

Rogue - Assassination

### Describe the Issue

Garrote is suggested multiple times in a row, target swapping is disabled so I suspect the priority thinks I am playing with Indiscriminated Carnage talent when I'm not.

### How to Reproduce

1. Enter game
2. standard raid build
3. hit aoe target dummies
4. wrong suggestions after first ambush

### Snapshot (Link)

https://pastebin.com/h1zpFSiR

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/mwUjAWe7tRa1ySttkEswrR

**Comment by Squatty1** [comment]
looking at the raidbot sim, its target swap that seems to cause the behaviour.

![image](https://github.com/user-attachments/assets/685b2a15-b68c-4172-8c7f-95c816261823)

**Comment by Hekili** [comment]
tl;dr -- next SimC update for the Assassin priority solves this issue.
